packages/front-end/pages/design-system/index.tsx

```diff
@@ -96,6 +98,56 @@ export default function DesignSystemPage() {
         </Flex>
       </div>
       <div className="appbox p-3">
+        <h3>Radio Group</h3>
```

Comment:
nitpick: Wrap this in its own `appbox` instead of sharing the one for HelperText

Reply:
was a mistake in resolving the merge conflict, sorry. fixed.